components/infobox/wikis/valorant/infobox_team_custom.lua

```diff
@@ -47,6 +47,7 @@ function CustomInjector:parse(id, widgets)
 end
 
 function CustomTeam:createBottomContent()
+	if not _team.args.disbanded then
 	return Template.expandTemplate(
 		mw.getCurrentFrame(),
 		'Upcoming and ongoing matches of',
```

Comment:
```suggestion
	if not _team.args.disbanded then
		return Template.expandTemplate(
			mw.getCurrentFrame(),
			'Upcoming and ongoing matches of',
```
But for all of it (github doesn't allow me to suggest the change some line 54-56 are out of scope according to it)